PROTOCOL.md

```diff
@@ -679,11 +755,22 @@ Refer to the [appendix](#checkpoint-schema) for an example on the schema of the
  - Data files MUST be uniquely named and MUST NOT be overwritten. The reference implementation uses a GUID in the name to ensure this property.
 
 ## Append-only Tables
-When the table property `delta.appendOnly` is set to `true`:
-  - New log entries MUST NOT change or remove data from the table.
-  - New log entries may rearrange data (i.e. `add` and `remove` actions where `dataChange=false`).
+Enablement:
+ - If the table is on a Writer Version starting from 2 up to 6, the table property `delta.appendOnly` must be set to `true`.
+ - If the table is on Writer Version 7, the feature `appendOnly` must exist in the table `protocol`'s `writerFeatures`, and the table property `delta.appendOnly` must be set to `true`.
```

Comment:
Can we factor this differently, to separate out the table property which is now duplicated? E.g.:

```suggestion
 - The table must be on a Writer Version between 2 and 7.
 - If the table is on Writer Version 7, the feature `appendOnly` must exist in the table `protocol`'s `writerFeatures`.
- The table property `delta.appendOnly` must be set to `true`.
```